tests/backup/backup_upgrade_test.go

```diff
@@ -655,6 +585,72 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching name of the first schedule backup with schedule [%s]", allNSScheduleName))
 			log.Infof("First schedule backup name: [%s]", firstScheduleBackupName)
 		})
+
+		Step("Initiate a go routine to stop CR backups for PVCs for partial backup validation", func() {
+			if IsPxInstalled() {
+				// Making a list of PVCs to fail
+				k8sCore := k8score.Instance()
+				pvcList, err := k8sCore.GetPersistentVolumeClaims(partialAppNamespaces[0], make(map[string]string))
+				log.FailOnError(err, fmt.Sprintf("error getting PVC list for namespace %s", partialAppNamespaces[0]))
+				log.Infof("pvc list is %v", pvcList)
+				for _, pvc := range pvcList.Items {
+					log.Infof("pvc %v", pvc)
+					failedVolumes = append(failedVolumes, &pvc)
+				}
+				if len(failedVolumes) > 0 {
+					partialAppContexts[0].SkipPodValidation = true
+				}
+				var wg sync.WaitGroup
+				var ctx context1.Context
+				ctx, cancelFunc = context1.WithCancel(context1.Background())
+				wg.Add(len(failedVolumes))
+				for _, pvc := range failedVolumes {
+					go func(pvc *corev1.PersistentVolumeClaim) {
+						defer GinkgoRecover()
+						defer wg.Done()
+						log.Infof("Starting go routine to stop the CR backups for namespace [%s] and pvc [%s]", pvc.Name, pvc.Namespace)
+						err := WatchAndStopCloudsnapBackup(pvc.Name, pvc.Namespace, 2*time.Hour, ctx)
+						if err != nil {
+							dash.VerifySafely(err, nil, fmt.Sprintf("watching and stopping cloudsnap backup for PVC [%s] in namespace [%s]: %v", pvc.Name, pvc.Namespace, err))
+						} else {
+							log.InfoD("Cloudsnap backup watch and stop process completed successfully for PVC [%s] in namespace [%s].", pvc.Name, pvc.Namespace)
+						}
+					}(pvc)
+				}
+			}
+		})
```

Comment:
Add an else block and mention that we are skipping this step since it is a non-px cluster

Reply:
Addressed